pulldown-cmark-mdcat/src/render/write.rs

```diff
@@ -360,3 +362,28 @@ pub fn write_start_heading<W: Write>(
         InlineAttrs { style, indent: 0 },
     ))
 }
+
+fn calculate_column_widths(table: &CurrentTable) -> Vec<usize> {
+    let mut widths = vec![0; table.rows.first().unwrap().cells.len()];
```

Comment:
Unwrap always sets off my alarm; would you add a small comment above this line to explain why we can safely call unwrap here without risking a panic?